### PR TITLE
Bugfix/1305 expansion test flakiness

### LINKED ISSUE #1305: Expansion test flakiness
**Component**: tests, salt

**What happened**:

The expansion test, introduced by #982, show multiple occurences of flaky failures:

- Pods from `monitoring` namespaced are expected to be scheduled on the new node, which may, or may not, be true
- Timeout issues when waiting for running Pods
- One occurence of unreachable target node (`Unable to connect to port 22 on <node-ip>`)

**What was expected**: No flakiness.

**Steps to reproduce**: N/A (flaky)

**Resolution proposal** (optional):

- Handle *infra* Pods differently than static Pods, w.r.t. their expected location
- Increase timeouts
- Add pre-checks (see #1056)

### PR DESCRIPTION
**Pull request by jbertran**
**Component**:

**Context**: 

Flaky test management time!

* Some pods fail to start even after 90s (30 x 3s wait).
Example build: https://eve.devsca.com/github/scality/metalk8s/#/builders/1/builds/6895

* More stuff...

**Summary**:

* Revert expansion test disable
* Change approach for command execution in salt-master
* Enable log collection on `node1`, since we now do expansion

**Acceptance criteria**: 

* No more flakies due warning logs going into `stderr` in salt output of `deploy_node`.
* Actual formula execution errors still report failures (see https://eve.devsca.com/github/scality/metalk8s/#/builders/10/builds/2300 for example, where a syntax error inserted in the `deploy_node` Jinja file caused the build to fail with comprehensive error output).

---

Closes: #1305

### CODE REVIEW

**Comment by jbertran** [comment]
@bert-e reset

File: tests/install/steps/test_expansion.py [hunk 1]
```diff
@@ -115,7 +115,7 @@ def kubectl_exec(
     if kwargs.get('namespace'):
         kube_args.extend(['-n', kwargs.get('namespace')])
 
-    kubectl_cmd_tplt = 'kubectl exec %s %s -- %s'
+    kubectl_cmd_tplt = 'kubectl exec {} {} -- {}'
```
Comment (slaperche-scality) [inline tests/install/steps/test_expansion.py:118]: This doesn't belong to this commit.

**Review by slaperche-scality — APPROVED** [review]

**Comment by jbertran** [comment]
@bert-e approve

**Comment by jbertran** [comment]
@bert-e reset
